ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-commands.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+TFVARS_PATH=/var/run/secrets/ci.openshift.io/cluster-profile/secret.auto.tfvars
+vsphere_user=$(grep -oP 'vsphere_user="\K[^"]+' ${TFVARS_PATH})
+vsphere_password=$(grep -oP 'vsphere_password="\K[^"]+' ${TFVARS_PATH})
+
+declare -a vips
+mapfile -t vips < "${SHARED_DIR}/vips.txt"
+
+cat >> "${CONFIG}" << EOF
+baseDomain: origin-ci-int-aws.dev.rhcloud.com
+platform:
+  vsphere:
+    cluster: devel
```

Comment:
@jcpowermac I will change `vcsa` -> `vcsa-ci` below. Do you know value for `cluster` here?

Reply:
That is correct as `devel`